docs-src/0.6/src/guides/fullstack/secrets.md

```diff
@@ -0,0 +1,70 @@
+# Managing Secrets
+
+In Dioxus fullstack applications, there are two types of secrets:
+
+1. **Server-side secrets**: These are sensitive values like database credentials or API keys that should never be exposed to the client.
+
+1. **Client-side secrets**: These are values needed by the client application at compile time, such as API keys.
+
+## Server-Side Secrets
+
+Server-side secrets are intended to remain confidential and are only accessible within the server environment. To manage these secrets:
+
+1. **Create a `.env` file**: At the root of your Dioxus project, create a `.env` file to store your server-side secrets. For example:
+
+   ```sh
+   MONGODB_DB_NAME=your_database_name
+   ```
+
+1. **Load the `.env` file in your server entry point**: Use the [`dotenv`](https://docs.rs/dotenv) crate to load the environment variables. In your `main` entry point:
+
+   ```rust
+   use dotenv::dotenv;
```

Comment:
dotenv [isn't maintained anymore](https://rustsec.org/advisories/RUSTSEC-2021-0141.html). Can we use std::env::var directly or an alternative library?